### PR TITLE
Reporting

### PR DESCRIPTION
**Pull request by brightoibe**
Checked for the existence of reporting_report_design before inserting, removed concept_answer_id inserts during concept_answer inserts, updated reporting_report_design for Maternal Cohort and Child Follow up, updated the RADET query to include valid/invalid columns.

### CODE REVIEW

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -94,11 +94,16 @@
...
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM concept_answer WHERE concept_id='164507' and answer_concept='162200'
+            </sqlCheck>
+        </preConditions>
         <insert tableName="concept_answer">
             <column name="concept_id" valueNumeric="164507"/>
             <column name="answer_concept" valueNumeric="162200"/>
             <column name="creator" valueNumeric="1"/>
-            <column name="date_created" valueDate="now()"/>
+            <column name="date_created" valueDate="2021-02-02"/>
```
Comment (wandechris) [inline api/src/main/resources/liquibase.xml:106]: Please change it to now(), this helps us know when the item was migrated

Comment (brightoibe) [reply api/src/main/resources/liquibase.xml]: This has been implemented.

Comment (brightoibe) [reply api/src/main/resources/liquibase.xml]: Do I need to do another pull request?

**Review by wandechris — APPROVED** [review]

File: api/src/main/resources/liquibase.xml (hunk 1 above)
Comment (wandechris) [reply api/src/main/resources/liquibase.xml]: No Bright, this one is fine, I've approved

**Pull request closed without merging**